src/curated-corpus/components/ScheduleSummaryLayout/ScheduleSummaryLayout.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { ScheduleSummaryCard } from '../ScheduleSummaryCard/ScheduleSummaryCard';
+import { getGroupedPublisherData } from '../../helpers/publishers';
+import { getDisplayTopic, getGroupedTopicData } from '../../helpers/topics';
+import { Maybe, ScheduledCorpusItem } from '../../../api/generatedTypes';
+
+interface ScheduleSummaryLayoutProps {
+  /**
+   * Scheduled items for a given date - to summarize in the component below.
+   */
+  scheduledItems: ScheduledCorpusItem[];
+}
+
+/**
+ * This component is responsible for taking a list of scheduled stories
+ * for a given date + surface combination and transforming it into two
+ * summarised, human-readable tables of data.
+ *
+ * @param props
+ * @constructor
+ */
+export const ScheduleSummaryLayout: React.FC<ScheduleSummaryLayoutProps> = (
+  props
+): JSX.Element => {
+  const { scheduledItems } = props;
+
+  // Extract all publishers from scheduled item data
+  const publishers =
+    scheduledItems.map(
+      (item: { approvedItem: { publisher: string } }) =>
+        item.approvedItem.publisher
+    ) ?? [];
+
+  // Extract all topics from scheduled item data
+  const topics =
+    scheduledItems.map(
+      (item: { approvedItem: { topic: Maybe<string> | undefined } }) =>
+        getDisplayTopic(item.approvedItem.topic)
+    ) ?? [];
```

Comment:
Potential for abstracting away into helper functions? 😄

Reply:
Yep, but so far these have only been used twice, let's wait for the third time :)